### PR TITLE
Updated a typo in the browserify guide

### PR DESCRIPTION
**Pull request by ellt92**
Just a very minor typo I picked up on when using this awesome repo. Confused me for a split second.

### CODE REVIEW

File: docs/guides/browserify.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ compatible with React 0.13 and React 0.14.
 Unfortunately, these conditional requires mean there is a bit of extra setup with bundlers like
 browserify.
 
-In your browserify configuration, you simply need to make sure that the following two files are
+In your browserify configuration, you simply need to make sure that the following three files are
```
Comment (ljharb) [inline docs/guides/browserify.md]: to prevent further inconsistencies, maybe just "following files"?

Comment (ellt92) [inline docs/guides/browserify.md]: Nice point, I've updated master with hat suggestion.

**Comment by ljharb** [comment]
@ellt92 thanks! any chance you could rebase these down to a single commit on latest master? nbd if not.

**Comment by ellt92** [comment]
Sure thing, just squashed them now.

**Pull request merged**